guides/v2.1/frontend-dev-guide/css-topics/using-fonts.md

```diff
@@ -44,6 +44,21 @@ To ensure the stability of your customizations and prevent upgrades from overwri
         font-style: normal
     }
     ```
+	Add Icon in specific class using mixin: 
```

Comment:
Hi @abrarpathan19.
This topic describes steps on how to use custom fonts in the system. The icons matter is described in the next topic. Please, move your changes to the corresponding topic.

Thank you.

Reply:
Hello @rogyar,
Thanks for reviewing my PR and suggestions,
I think there is not the corresponding topic regarding the Icon, I am unable to find out that. 

please let know where to place this code, as per my understanding this is the right place to add this topic.

we can also create a new topic for the Icon, Kindly advice me.